.travis/publish.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly tag="${TRAVIS_TAG}"
+readonly blobaccount="${DEPLOY_BLOB_ACCOUNT_NAME}"
+readonly blobkey="${DEPLOY_BLOB_ACCOUNT_KEY}"
+readonly blobcontainer="${DEPLOY_BLOB_CONTAINER}"
+readonly blobname="fortis-${tag}.jar"
+
+log() {
+  echo "$@" >&2
+}
+
+check_preconditions() {
+  if [ -z "${tag}" ]; then
+    log "Build is not a tag, skipping publish"
+    exit 0
```

Comment:
What's your opinion on creating a fat jar for every commit instead of just tags?